index.js

```diff
@@ -19,7 +19,10 @@ module.exports = function(source) {
 
     const callback = this.async(),
         options = Object.assign({}, this.options.bemLoader, loaderUtils.getOptions(this)),
-        levelsMap = options.levels || bemConfig.levelMapSync(),
+        setLevels = options.sets && bemConfig.levelsSync(options.sets) || [],
```

Comment:
Could you describe desired behaviour
and add it to README section with options

Reply:
I think we should add examples inside `sdk.config` and refer to them

Reply:
in my opinion, we should add describe to README, and also add example inside sdk.config